packages/core/src/spec/lib/androidSdk/AndroidSdkToolsSpec.ts

```diff
@@ -92,13 +91,14 @@ describe('AndroidSdkTools', () => {
     ];
 
     tests.forEach((test) => {
-      it(`Sets the correct ANDROID_HOME on ${test.platform}`, () => {
+      it(`Sets the correct ANDROID_HOME on ${test.platform}`, async () => {
         spyOn(fs, 'existsSync').and.returnValue(true);
         const config = buildMockConfig(test.platform);
         const process = buildMockProcess(test.platform);
         const jdkHelper = new JdkHelper(process, config);
         const mockLog = new MockLog();
-        const androidSdkTools = new AndroidSdkTools(process, config, jdkHelper, mockLog);
+        const androidSdkTools = (await AndroidSdkTools.create(process, config,
+            jdkHelper, mockLog));
```

Comment:
This parenthesis is unnecessary. We can remove it and get a line with 99 charts.
```suggestion
        const androidSdkTools = await AndroidSdkTools.create(process, config, jdkHelper, mockLog);
```